test/test_plugin.py

```diff
@@ -511,19 +511,19 @@ def test_require_bot_privilege_private_message(configfactory,
     def mock(bot, trigger):
         return True
 
-    assert mock(bot, bot._trigger) is True
+    assert mock(bot, bot._trigger) is not True
```

Comment:
This assertion seems like it should have an associated message with this change.